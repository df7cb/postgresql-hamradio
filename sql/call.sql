SELECT 'DF7CB'::call;
SELECT 'DF7CB/P'::call;
SELECT 'F/DF7CB/P'::call;
SELECT 'DE-1234'::call; -- SWL

-- errors
SELECT 'df7cb'::call;
SELECT 'DE 1234'::call;
