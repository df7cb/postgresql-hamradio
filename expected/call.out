SELECT 'DF7CB'::call;
 call  
-------
 DF7CB
(1 row)

SELECT 'DF7CB/P'::call;
  call   
---------
 DF7CB/P
(1 row)

SELECT 'F/DF7CB/P'::call;
   call    
-----------
 F/DF7CB/P
(1 row)

SELECT 'DE-1234'::call; -- SWL
  call   
---------
 DE-1234
(1 row)

-- errors
SELECT 'df7cb'::call;
ERROR:  value for domain call violates check constraint "valid_callsign"
SELECT 'DE 1234'::call;
ERROR:  value for domain call violates check constraint "valid_callsign"
